challenges/README.md

```diff
@@ -70,6 +74,13 @@ same as the users object on Users Api
 
 Return the same response from [GET /challenges](#get-challenges)
 
+
+- **Error Response:**
+  - **Code:** 404
+    - **Content:** `{"statusCode":404,"error":"Not Found","message":"Unable to add challenge"}`
+  - **Code:** 503
+    - **Content:** `{"statusCode":503,"error":"Service Unavailable","message":"Something went wrong. Please try again or contact admin"}`
```

Comment:
Can we indent the JSON for all responses, please?

Reply:
Yaa sure, I will do that